addons/event/views/event_tag_views.xml

```diff
@@ -58,7 +58,7 @@
                 <tree string="Event Tags Categories">
                     <field name="sequence" widget="handle"/>
                     <field name="name"/>
-                    <field name="category_id"/>
+                    <field name="category_id" options="{'no_quick_create':True}"/>
```

Comment:
Why disable quick create entirely? Does setting `default_website_id` not work with quick create?

Reply:
yes, it had problem. when you type new tag name it suggests two options: 'create and edit' and 'create'. For some reason context wasn't passed when i selected 'create' option. (it didn't set website on category)
![image](https://github.com/odoo/odoo/assets/52849166/488e111b-b1ad-4716-a4d9-b9d3d85d0e9d)

Reply:
Mm ok, not much we can do then I guess. Not 100% sure it's worth the added clicks but we'll see what lna says